manim/mobject/graph.py

```diff
@@ -158,6 +158,26 @@ def construct(self):
                                        (4, 7): {"stroke_color": RED}})
                 self.add(g)
 
+    You can also lay out a partite graph on columns by specifying
+    a list of the vertices on each side and choosing the partite layout.
+
+    .. note::
+
+        All of the vertices which are not in one of the sides you provide
+        will be put into another column.
+
+    .. manim:: PartiteGraph
+        :save_last_frame:
+        import networkx as nx
```

Comment:
The example requires a newline between the head of the directive and the code block.

```suggestion

        import networkx as nx
```